src/elexmodel/handlers/data/Featurizer.py

```diff
@@ -9,7 +9,18 @@ class Featurizer(object):
 
     def __init__(self, features, fixed_effects):
         self.features = features
-        self.fixed_effects = fixed_effects
+        if isinstance(fixed_effects, list):
+            self.fixed_effect_cols = fixed_effects
+            self.fixed_effect_params = {fe: ["all"] for fe in fixed_effects}
+        else:
+            self.fixed_effect_cols = list(fixed_effects.keys())
+            self.fixed_effect_params = {}
+            for fe, params in fixed_effects.items():
+                if params == "all":
```

Comment:
is this in case someone inputs the parameter as:
```
{'postal_code': 'all'}
```
instead of
```
{'postal_code': ['all']}
```

Reply:
Yes